app/Console/Commands/GenerateArticle.php

```diff
@@ -207,7 +207,8 @@ protected function getExistingWebsite($websites, $websiteDomain)
     protected function collectAuthorInfo()
     {
         $enteredAuthorName = $this->ask("Enter author name?");
-        $enteredAuthorTwitter = $this->ask("Enter author's twitter handle (without '@').");
+        $enteredAuthorTwitter = $this->ask("Enter author's twitter handle (without '@').", false);
+        $enteredAuthorTwitter = $enteredAuthorTwitter == false ? null : $enteredAuthorTwitter;
```

Comment:
OOPs no.

We might have author already added in our database. In this case, this code will remove twitter account for existing author. We should do that as follow:

Based on author name, check if author already exist in database.

- if exist:
    - take take twitter account from database
- If not exist:
    - ask twitter account. Here user may skip twitter account.